src/web/common/Countdown.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+
+function Countdown({ callback, className, timeInSeconds, paused }) {
+  const [currentTime, setCurrentTime] = useState('00:00');
+  let logInterval;
+  let duration = timeInSeconds;
+
+  const startCountdown = () => {
+    let minutes;
+    let seconds;
+
+    logInterval = setInterval(() => {
```

Comment:
Having a long timeout here, I wonder how this is going to work if the user switches tabs or windows in their computer. Can we still rely this timer will still be somewhat accurate? I know in the past there were throttling issues in some cases.